framework/Models/PostProcessors/__init__.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  The Models module for building, running, and simulating things in RAVEN.
+
+  Created on May 9, 2017
+  @author: maljdp
+"""
+from .PostProcessorInterface import PostProcessorInterface
+from .FTImporter import FTImporter
+from .BasicStatistics import BasicStatistics
+from .LimitSurface import LimitSurface
+from .Metric import Metric
+from .ETImporter import ETImporter
+from .DataMining import DataMining
+from .SafestPoint import SafestPoint
+from .ValueDuration import ValueDuration
+from .SampleSelector import SampleSelector
+from .ImportanceRank import ImportanceRank
+from .CrossValidation import CrossValidation
+from .LimitSurfaceIntegral import LimitSurfaceIntegral
+from .FastFourierTransform import FastFourierTransform
+from .ExternalPostProcessor import ExternalPostProcessor
+from .InterfacedPostProcessor import InterfacedPostProcessor
+from .TopologicalDecomposition import TopologicalDecomposition
+from .DataClassifier import DataClassifier
+from .ComparisonStatisticsModule import ComparisonStatistics
+from .RealizationAverager import RealizationAverager
+from .ParetoFrontierPostProcessor import ParetoFrontier
+from .MCSimporter import MCSImporter
+from .EconomicRatio import EconomicRatio
+## These utilize the optional prequisite library PySide, so don't error if they
+## do not import appropriately.
+additionalModules = []
+try:
+  from .TopologicalDecomposition import QTopologicalDecomposition
+  from .DataMining import QDataMining
+  additionalModules.append(QTopologicalDecomposition)
+  additionalModules.append(QDataMining)
+except ImportError:
```

Comment:
please see PR #1503. These double imports are not needed. The only imports needed are in the Factory.py (it was supposed to be addressed in the PR #1480)

Reply:
I think that observation came after that PR, right? Either way, I concur, I like the idea that classes are only available through the factories. This may require some standardization so we can check types consistently; previously we would import the module and check instance, such as
```python
from Samplers import MonteCarlo
if isinstance(mySampler, MonteCarlo):
  ...
```
This might be harder with factories, but should be standardizable without checking against strings.

Reply:
I have cleaned up the __init__.py file